synpress.config.js

```diff
@@ -33,12 +39,12 @@ module.exports = defineConfig({
     testIsolation: false,
     setupNodeEvents,
     baseUrl: 'http://localhost:3000',
-    specPattern: 'tests/e2e/specs/**/*.{js,jsx,ts,tsx}',
+    specPattern: specPattern[process.env.EXTENSION],
     supportFile,
   },
   component: {
     setupNodeEvents,
-    specPattern: './**/*spec.{js,jsx,ts,tsx}',
```

Comment:
pattern is different here

Reply:
Yes. That's because spec files were mainly placed in e2e folder. There wasn't any individual component spec files in the repo.